test/helpers/add-pr-late-label.test.ts

```diff
@@ -0,0 +1,145 @@
+/*
+Copyright 2021 Expedia, Inc.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+  https://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { LATE_REVIEW } from '../../src/constants';
+import { addPrLateReviewLabels } from '../../src/helpers/add-pr-late-label';
+import { octokit } from '../../src/octokit';
+
+jest.mock('@actions/core');
+jest.mock('@actions/github', () => ({
+  getOctokit: jest.fn(() => ({
```

Comment:
Nitpick, but can we put the mocks in the actual test block where they are needed instead of all at the top of the file? It's hard to follow what the mocked return values are for each test

Reply:
Moved what we could, for some reason (although docs say it should be fine) if the original jest.mock's are not above the import it does not work